charts/backstage/templates/pvc.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: {{ printf "%s-audit-log" .Release.Name }}
+spec:
+  capacity:
+    storage: 5Gi
+  accessModes:
+    - ReadWriteOnce
+  persistentVolumeReclaimPolicy: Retain
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: {{ printf "%s-audit-log" .Release.Name }}
+spec:
```

Comment:
I think the storage class should at least be configurable from the `values.yaml` file. In https://issues.redhat.com/browse/RHIDP-2705, there was a request to make this configurable for the dynamic plugins volume in environments with no default storage class available.

Reply:
Can we leave that for another pull request and I will update both this one and the dynamic plugins volume?